File: codeforces/1A/1A.py
# !/usr/bin/pyhon3
# -*- coding: utf-8 -*-
"""
    A. Theatre Square
    auguralpha
    2020-03-27
"""

from math import ceil


def solution():
    print(ceil(n / a) * ceil(m / a))


if __name__ == '__main__':
    n, m, a = map(int, input().split())
    solution()

File: codeforces/1A/1A.go
// A. Theatre Square
// auguralpha
// 2020-03-27

package main

import "fmt"

func main() {
	var n, m, a uint64
	_, _ = fmt.Scanf("%d%d%d", &n, &m, &a)
	fmt.Println(((n + a - 1 ) / a) * ((m + a - 1) / a))
}
